src/commands/docs.ts

```diff
@@ -14,21 +13,24 @@ const Docs: Command = {
         .setRequired(true)
         .addChoices(...ResourceChoices)
     ),
-  run: async (interaction) => {
-    const resource = interaction.options.getString('resource');
-    const docsEmbed: MessageEmbedOptions = {
-      title: 'Documentation',
-      description: 'Please read the documentation thoroughly and carefully.',
-      color: 'GREEN',
-      fields: [
-        {
-          name: resource || 'Website',
-          value: `${DocsUrl}/${resource}`,
-        },
-      ],
-    };
-    return await interaction.reply({ embeds: [docsEmbed] });
+  async execute(interaction: CommandInteraction) {
+    const resourceOption = interaction.options.get('resource');
+    const resource = resourceOption?.value as string;
+    await sendDocumentationEmbed(interaction, resource);
   },
 };
 
+const sendDocumentationEmbed = async (interaction: CommandInteraction, resource: string) => {
+  const docsEmbed = new EmbedBuilder()
+    .setTitle('Documentation')
+    .setDescription('Please read the documentation thoroughly and carefully.')
+    .setColor(0x26e0a5)
```

Comment:
Does color not support strings? Would rather that than this weird hex value that you can't tell what it is off of the first glace.

Reply:
This is due to v14 stuff
https://discordjs.guide/popular-topics/embeds.html#using-the-embed-constructor